script.favourites/lib/favourites.py

```diff
@@ -39,7 +39,8 @@ def _parse_argv(self, params):
     def _get_favs(self):
         data = xbmc.executeJSONRPC('{"jsonrpc":"2.0", "method":"Favourites.GetFavourites", "params":{"properties":["window", "windowparameter", "thumbnail", "path"]}, "id":1}')
         favs = json.loads(data)
-        return favs['result']['favourites']
+        if 'result' in favs:
+            return favs['result']['favourites']
```

Comment:
Probably it makes sense to have an else block returning an empty list (depending on how is the callee)

Reply:
Ok it's called on a single location and it checks the return value (which in this case will be None)